tests/Agent/NewRelic.Testing.Assertions/NewRelic.Testing.Assertions.csproj

```diff
@@ -1,16 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;net7.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="coverlet.collector" Version="6.0.0">
```

Comment:
Why was this reference removed?

Reply:
Because this isn't a unit test project; it's a utility project.